c2tc-mobile/components/PhoneButton.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import Icon from "react-native-vector-icons/FontAwesome";
+import call from "react-native-phone-call";
+
+const buttonColor = "#424242";
+
+export default class PhoneButton extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      type: this.props.type,
+      number: this.props.number
+    };
+  }
+
+  getType() {
+    return this.state.type;
+  }
+
+  _onPressCall = () => {
+    const args = {
+      number: this.state.number,
+      prompt: false
+    };
+    call(args).catch(console.error);
```

Comment:
Rather than printing it to console, send an alert to the user.

Reply:
I don't think that the user should exactly get the full error - it should be printed to the console for debugging and then we can alert the user that there was some sort of general error instead